src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AtclauseOrderCheckTest.java

```diff
@@ -173,4 +173,36 @@ public void testAtclauseOrderLotsOfRecords() throws Exception {
         verify(checkConfig, getNonCompilablePath("InputAtclauseOrderLotsOfRecords.java"), expected);
     }
 
+    @Test
+    public void testAtclause() throws Exception {
+        final String tagOrder = "[@author, @version, @param, @return, @throws, @exception, @see,"
+                + " @since, @serial, @serialField, @serialData, @deprecated]";
```

Comment:
this is not default compare with https://checkstyle.org/config_javadoc.html#AtclauseOrder_Properties

message suppose to print expected order:
https://github.com/checkstyle/checkstyle/blob/bf86a3ad249ce60fdf90a433d5e9f739e56a9cca/src/main/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/messages.properties#L1

please recheck what is root of problem, there might be outdated value in html, please investigate and lets create  separate issue on this.

Reply:
Issue created in #10086